File: charArrayAndString/turkishEnglish.cpp
#include<iostream>
#include<cstdlib>
#include<cstring>
using namespace std;
int main(){
	char tr[][10] = {"Pazatesi", "Sali", "Carsamba", "Persembe", "Cuma",
		"Cumartesi", "Pazar"};
	char en[][10] = {"Monday", "Tuesday", "Wednesday", "Thursday", "Friday",
		"Saturday", "Sunday"};
	cout << "Enter a turkish name = ";
	char *word;
	cin >> word;
	int decision = 1;
	int i;
	for(i = 0; i < 7 && decision != 0; i++){
		decision = strcmpi(word, tr[i]);
	}
	cout << "English name = " << en[i - 1];
	return 0;
}

File: charArrayAndString/turkishEnglish.java
package springIntro;

import java.util.*;

import java.lang.*;

public class turkishEnglish {

	public static void main(String[] args) {
		Scanner in = new Scanner(System.in);
		String tr[] = {"Pazartesi", "Sali", "Carsamba", "Persembe", "Cuma",
				"Cumartesi", "Pazar"};
		String en[] = {"Monday", "Tuesday", "Wednesday", "Thursday", "Friday",
				"Saturday", "Sunday"};
		System.out.println("Enter a turkish name");
		String word = in.next();
		System.out.printf("English name = ");
		boolean decision = false;
		int i;
		for(i = 0; i < 7 && decision == false; i++) {
			decision = tr[i].equalsIgnoreCase(word);
		}
		System.out.println(en[i - 1]);
	}

}
